src/modules/csm/csm.py

```diff
@@ -219,6 +220,14 @@ def calculate_distribution(self, blockstamp: ReferenceBlockStamp) -> tuple[int,
                     # is not presented in the aggregates (e.g. exited, pending for activation etc).
                     continue
 
+                if v.validator.slashed:
```

Comment:
Move it before try..catch block.

Reply:
we can't do that. otherwise we will remove good operators that had slashed validator in the previous frame, for example